<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainerServiceDialog</class>
 <widget class="QDialog" name="TrainerServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Покупка тренерской услуги</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="serviceTypeLabel">
       <property name="text">
        <string>Вид тренера:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="serviceTypeComboBox">
       <item>
        <property name="text">
         <string>Тренажерный зал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Групповые тренировки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Плавание</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бокс</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Срок услуги:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="durationComboBox">
       <item>
        <property name="text">
         <string>Разовая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Неделя</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Месяц</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Полгода</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Год</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Два года</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Дата начала:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="startDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2025</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="costLabel">
       <property name="text">
        <string>Цена:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="costLabelValue">
       <property name="text">
        <string>500 руб.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrainerServiceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrainerServiceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>